.github/ISSUE_TEMPLATE/dotnet-breaking-change.md

```diff
@@ -95,3 +95,4 @@ assignees: ''
 #### Issue metadata
 
 * Issue type: breaking-change
+* Issue type: doc-idea
```

Comment:
I don't think this change alone is enough.

The config for the bot needs to be updated too:

https://github.com/dotnet/docs/blob/9eee581ebb8b94e8e5cde240ee2a59f74d96442e/.ghal.rules.json#L172-L176